### DISCUSSION TITLE
Dark (or other) Themes?

### DISCUSSION THREAD

**Discussion by grandevel** [segmentio/evergreen#1216] [General] [answered]
Are we expecting themes other than classic and default to be provided within evergreen? I am wondering if my team should invest time in creating a dark theme and other themes (such as accessible high-vis, etc) or whether you already have these in the works?

**Comment by akleiner2** [comment, marked as answer]
@grandevel Dark mode is definitely something that's on our roadmap, but there are potentially a couple of infrastructure-y things that we'd like to do before getting there. You're more than welcome to use the existing theming architecture to build a dark mode for your team / product, but I imagine that ours (what Evergreen will provide out-of-the-box) will be informed by the specific needs for our app(s) internally. We haven't started on it yet, but when we do - we'll let folks know about it in here. cc @kamebkj
